src/app/features/faq/faq.page.ts

```diff
@@ -31,11 +32,14 @@ export class FaqPage {
       .pipe(
         tap(event => {
           const postMessageEvent = event as MessageEvent;
-          if (postMessageEvent.data === 'iframe-on-load') {
-            this.iframeLoaded$.next(true);
-          }
-          if (postMessageEvent.data === 'iframe-back-button-clicked') {
-            this.navController.pop();
+          const data = postMessageEvent.data as BubbleToIonicPostMessage;
+          switch (data) {
+            case BubbleToIonicPostMessage.IFRAME_ON_LOAD:
+              this.iframeLoaded$.next(true);
+              break;
+            case BubbleToIonicPostMessage.IFRAME_BACK_BUTTON_CLICKED:
+              this.navController.back();
+              break;
           }
         }),
         untilDestroyed(this)
```

Comment:
**ChatGPT response** for `git show a9e4232dfd700d07b6a61ec73910927941b129ae src/app/features/faq/faq.page.ts `

The code looks good. Since it is a similar change to the previous one, it might be good to extract the switch statement into a separate function that can be reused across different components to avoid duplicating code.

Reply:
@tammyyang regarding ChatGPT response :

No need to extract to separate methods since all of them one liners